test/src/lib/backoff.spec.ts

```diff
@@ -0,0 +1,163 @@
+import { expect } from 'chai';
+
+import Failer from '../../lib/failer';
```

Comment:
Maybe is better to define the `Failer` object within this same file. If we need it somewhere else we can always refactor. Storing it in lib provides no immediate benefit imho

Reply:
moved failer into backoff tests